api/src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_app_gdpr_user_extract.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+import logging
+
+from pcapi.core.users import factories as users_factories
+from pcapi.core.users import models as users_models
+
+
+logger = logging.getLogger(__name__)
+
+
+def create_gdpr_user_extract_data() -> None:
+    logger.info("create_gdpr_user_extract_data")
+
+    author_query = users_models.User.query.filter(users_models.User.has_admin_role).limit(3).all()
+    beneficiary_query = users_models.User.query.filter(users_models.User.has_beneficiary_role).limit(2).all()
+    underage_beneficiary_query = users_models.User.query.filter(users_models.User.has_underage_beneficiary_role).first()
+
```

Comment:
```suggestion
    author_query = users_models.User.query.filter(users_models.User.has_admin_role).order_by(users_models.User.id).limit(3).all()
    beneficiary_query = users_models.User.query.filter(users_models.User.has_beneficiary_role).order_by(users_models.User.id).limit(2).all()
    underage_beneficiary_query = users_models.User.query.filter(users_models.User.has_underage_beneficiary_role).order_by(users_models.User.id).first()

```
Ajouter un order by permettra d'avoir une sandbox plus stable qui generera toujours les mêmes données.